<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassifierDlibSvmNuTrain</class>
 <widget class="QWidget" name="ClassifierDlibSvmNuTrain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_13">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Binary classification label:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="selectedLabel">
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="selectSlices">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Select slices</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="bottomSliceLimitLabel">
         <property name="text">
          <string>Bottom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="bottomSliceLimit">
         <property name="text">
          <string>88</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="topSliceLimitLabel">
         <property name="text">
          <string>Top</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="topSliceLimit">
         <property name="text">
          <string>89</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="setAutomatically">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Set automatically</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QLabel" name="nuLowLimitLabel">
             <property name="text">
              <string>Nu low limit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nuLowLimit">
             <property name="text">
              <string>0.01</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="nuHighLimitLabel">
             <property name="text">
              <string>Nu high limit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nuHighLimit">
             <property name="text">
              <string>0.2</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="stepsLabel">
             <property name="text">
              <string>Steps</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="steps">
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="gammaLowLimitLabel">
             <property name="text">
              <string>γ low [10^]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="gammaLowLimit">
             <property name="text">
              <string>-3</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="gammaHighLimitLabel">
             <property name="text">
              <string>γ high [10^]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="gammaHighLimit">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QLabel" name="gammaStepsLabel">
             <property name="text">
              <string>Steps</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="gammaSteps">
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="gammaLabel">
             <property name="text">
              <string>γ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="nuLabel">
             <property name="text">
              <string>ν</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLineEdit" name="gamma">
             <property name="text">
              <string>0.001</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nu">
             <property name="text">
              <string>0.05</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
